lib/cli/interactive-setup/utils.js

```diff
@@ -54,7 +54,12 @@ module.exports = {
       isInServiceContext: Boolean(serviceDir),
       isLoggedIntoDashboard: isAuthenticated(),
       hasLocalAwsCredentials: hasLocalCredentials(),
-      isDashboardEnabled: Boolean(configuration && configuration.org && configuration.app),
+      isDashboardEnabled: Boolean(
+        configuration &&
+          configuration.org &&
+          configuration.app &&
+          (!configuration.console || configuration.dashboard)
```

Comment:
Is that check needed? Isn't having `app` defined is enough here?

Reply:
Not exactly, as `configuration.console` disables _dashboard_ support, unless `dashboard: true` is set.